packages/lb-annotation/src/core/pointCloud/cache.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file Cache the data that PointCloud can share
+ * @createdate 2022-07-26
+ * @author Ron <ron.f.luo@gmail.com>
+ */
+
+import { PCDLoader } from './PCDLoader';
+
+export class PointCloudCache {
+  public pcdLoader: PCDLoader;
+
+  private pointsMap: Map<string, THREE.Points>;
+
+  private static instance: PointCloudCache;
+
+  private constructor() {
+    this.pcdLoader = new PCDLoader();
+    this.pointsMap = new Map();
+  }
+
+  public static getInstance() {
+    if (this.instance == null) {
+      this.instance = new PointCloudCache();
+    }
+    return this.instance;
+  }
+
+  public loadPCDFile = (src: string) => {
```

Comment:
Add GC scheduler to prevent memory exceed

Reply:
size? 10 mb * 50